README.md

```diff
@@ -25,27 +25,35 @@ If you are running into issues with this app, please drop a mail to [support@ent
 - [AUR](https://aur.archlinux.org/packages/ente-desktop-appimage):
   `yay -S ente-desktop-appimage`
 
-## Develop
+## Building from source
 
-To clone and run this repository you'll need [Git](https://git-scm.com) and [Node.js](https://nodejs.org/en/download/) (which comes with [npm](http://npmjs.com)) installed on your computer. From your command line:
+You'll need to have node installed on your machine. e.g. on macOS you can do
+`brew install node`. After that, you can run the following commands to fetch and
```

Comment:
should we also mention yarn, it's not by default installed with node and has to be installed separately https://classic.yarnpkg.com/lang/en/docs/install/#mac-stable

Reply:
It comes with node on brew. But I can mention it too.

Reply:
Added!